api/v1alpha1/providertemplate_types.go

```diff
@@ -15,24 +15,82 @@
 package v1alpha1
 
 import (
+	"fmt"
+
+	"github.com/Masterminds/semver/v3"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// ChartAnnotationCAPIVersion is an annotation containing the CAPI exact version in the SemVer format associated with a ProviderTemplate.
+const ChartAnnotationCAPIVersion = "hmc.mirantis.com/capi-version"
+
 // ProviderTemplateSpec defines the desired state of ProviderTemplate
 type ProviderTemplateSpec struct {
-	TemplateSpecCommon `json:",inline"`
+	Helm HelmSpec `json:"helm"`
+	// Compatible CAPI provider version set in the SemVer format.
+	CAPIVersion string `json:"capiVersion,omitempty"`
+	// Represents required CAPI providers with exact compatibility versions set. Should be set if not present in the Helm chart metadata.
```

Comment:
these are providers that exposed/enabled when the template is installed

Reply:
Should this comment be related to the `spec`? `Exposed` is already in-use in the `status` field